.husky/pre-commit

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
-
-yarn lint-staged
-forge snapshot --check --match-contract Gas
\ No newline at end of file
+forge snapshot --check --match-contract Gas
+forge fmt --check
```

Comment:
does this lint all files or just the files being staged? 

from an initial read it looks like we are keeping lint staged installed, but we're running these checks on all the files. 

ideally, we'd just run the linter on the files being comitted.

Reply:
After some local testing, I can confirm that this would have ensured that `forge fmt --check` is applied to all files.

In the most recent commit, I utilized `lint-staged` program to ensure that `forge fmt --check` is only applied to staged files.

I tested it locally as well!

Local test example: 
- In the first screenshot, I stage only one of the changed files `src/BundleResistry.sol`.
- In the second screenshot, `forge fmt --check` runs only on the staged file `src/BundleRegistry.sol` when I try committing.
![image](https://user-images.githubusercontent.com/33789105/208137173-0ea070ab-0394-447b-a7d5-51f21353a9f1.png)
![image](https://user-images.githubusercontent.com/33789105/208137550-8896a538-4bab-4961-9b62-28c636ba522b.png)